Neos.ContentRepository/Classes/Domain/Service/PublishingService.php

```diff
@@ -274,11 +275,12 @@ public function emitNodePublished(NodeInterface $node, Workspace $targetWorkspac
      * The signal emits the node that has been discarded.
      *
      * @param NodeInterface $node
+     * @param Workspace|null $baseWorkspace
      * @return void
      * @Flow\Signal
      * @api
      */
-    public function emitNodeDiscarded(NodeInterface $node)
+    public function emitNodeDiscarded(NodeInterface $node, ?Workspace $baseWorkspace = null)
```

Comment:
fyi due to this this change needs a proxy cache flush as the Neos Neos `PublishingService` must be rebuild.